torch/_inductor/graph.py

```diff
@@ -50,6 +50,17 @@ def supported_dtype_of_cpp_wrapper(dtype):
     return dtype in supported_dtype
 
 
+def supported_extern_kernel_of_cpp_wrapper(buffer):
+    for kernel_type in [
+        ir.MatrixMultiply,
+        ir.BatchMatrixMultiply,
+        ir.MatrixMultiplyAdd,
+    ]:
+        if isinstance(buffer, kernel_type):
+            return True
+    return False
```

Comment:
```suggestion
    return isinstance(buffer, (ir.MatrixMultiply, ir.BatchMatrixMultiply, ir.MatrixMultiplyAdd,))
```
Also, maybe invoking `isinstance` directly from `check_buffer_for_cpp_wrapper` is simpler?

Reply:
Changed to directly invoking `isinstance` from `check_buffer_for_cpp_wrapper`.